ethereum/contracts/Campaign.sol

```diff
@@ -28,9 +28,7 @@ contract Campaign {
     }
 
     function contribute() public payable {
-        require(msg.value > minimumContribution);
-
-        if (approvers[msg.sender] != true) {
+        if (approvers[msg.sender] != true && msg.value > minimumContribution) {
```

Comment:
I think this is all that needed to be changed in the contract... much easier than I thought.